docs/COMMIT_MESSAGES.md

```diff
@@ -1,6 +1,6 @@
 # How to write a commit message
 
-Any commit needs a helpful message. Mind the following guidelines when committing to any of OMaps repositories at GitHub.
+Any commit needs a helpful message. Mind the following guidelines when committing to any of Organic Maps repositories at GitHub.
 
 1. Separate subject from body with a blank line.
 2. Limit the subject line to **50 characters** or less.
```

Comment:
50 characters is not a hard limit, just a rule of thumb. Keeping subject lines at this length ensures that they are readable, and forces the author to think for a moment about the most concise way to explain what’s going on.

Tip: If you’re having a hard time summarizing, you might be committing too many changes at once. Strive for atomic commits (a topic for a separate post).

GitHub’s UI is fully aware of these conventions. It will warn you if you go past the 50 character limit:

![image](https://user-images.githubusercontent.com/1799054/117261117-73516000-ae58-11eb-86f8-6547c0aa3813.png)

And will truncate any subject line longer than 72 characters with an ellipsis:

So shoot for 50 characters, but consider 72 the hard limit.

Reply:
Обновил

Reply:
72 значит отличное число для лимита. Не люблю пушить такие правила, а потом догадываться, что имел ввиду автор коммита, которому не хватило букв для объяснения.

Reply:
ну вот гитхаб говорит, что 50 лучше, чем 72.